mp-SuggestionCard

MarcosPauloRTK opened the pull request:
Componente SuggestionCard criado e estilizado, aguardando review.

LuanRethink committed:
starRating component

vinicius-delmo committed:
feat(filter): Add product filter select component

LuanRethink committed:
removed unecessary dependency

LuanRethink committed:
changed the style flow

LuanRethink committed:
removed dependency

LuanRethink committed:
Merge pull request #5 from leticiavargas/LB-starRating

starRating component

LuanRethink committed:
card component

LuanRethink committed:
removendo reuso de css

LuanRethink committed:
Merge branch 'LB-starRating' into lb-card-component

LuanRethink committed:
removed top

LuanRethink committed:
aligned at center

LuanRethink committed:
changed star size and gap

LuanRethink committed:
changed div size

LuanRethink committed:
moved the content to start,

LuanRethink committed:
already using alig-items

LuanRethink committed:
centralized component

LuanRethink committed:
onclick function

LuanRethink committed:
removed onClick

LuanRethink committed:
alt changed

LuanRethink committed:
width changed

LuanRethink committed:
alt text changed

vinicius-delmo committed:
refactor:
changing attributes, importing fonts and CSS styling

LuanRethink committed:
swiapable added

LuanRethink committed:
added roboto font

LuanRethink committed:
price formation changed

LuanRethink committed:
title changed to tag h2

mhid-dev committed:
Merge branch 'main' into showcase

mhid-dev committed:
md-ProductCardImage v1 (#10)

* md-ProductCardImage v1

* To-Do lógica botão

* To-Do logica botão de compartilhar

* Atualizações do code review

mhid-dev committed:
Removendo header do App.js

mhid-dev committed:
Merge branch 'showcase' of https://github.com/leticiavargas/naturelle into showcase

LuanRethink committed:
first component

LuanRethink committed:
alt

LuanRethink committed:
changed indicator styles

LuanRethink committed:
Merge branch 'showcase' into lb-card-component

vinicius-delmo committed:
Merge pull request #11 from leticiavargas/lb-card-component

Lb card component

mhid-dev committed:
Merge branch 'main' into showcase

vinicius-delmo committed:
feat:
implementation of the product search field

vinicius-delmo committed:
adding folder with images

LuanRethink committed:
removed useless css

LuanRethink committed:
removed image importation

MarcosPauloRTK committed:
SuggestionCard created

Component SuggestionCard created and styled

LuanRethink commented:
Salve Marcos,
Percebi que a cor de background está diferente, deveria ser #F0E7CF.
Acredito que esteja faltando a fonte Roboto do componente.
Parece que a distancia do shoppingCard está um pouco menor da lateral.

Mas está muito bom!

vinicius-delmo commented:
Código está muito bom, parabéns.
Além do que o Luan falou só tenho duas pequenas observações:
1 - No JSX acho que não é necessário o uso do react fragment "<> </>" pois o códido já está envolto na DIV .SuggestionCard 
2 - No botão a função está com um erro de sintaxe, "onClink" -> "onClick"

leticiavargas commented on src/components/ProductCardImage/ProductCardImage.jsx:
```diff
@@ -0,0 +1,65 @@
+        </button>
+      )}
+      <div className={`ProductCardImageLabels ${labelContainerClass}`}>
+        {hasMilk && (
```
gostei da lógica ... mas pensando em linguagem, é um pouco estranho ler "temLeite" e daí mostrar uma tag de Zero Lactose ... ou então um "tem açucar" e a tag ser Zero açucar

leticiavargas commented on src/components/productCard/index.jsx:
```diff
@@ -0,0 +1,26 @@
+          {name} | {weigth}
+        </h2>
+        <p className="cardRating">
+          <CustomizedRating rating={rate} precision={0.1}></CustomizedRating>{" "}
```
iiiii ... esses espaços a moda gambiarra não dá não heim ... nada de {"  "} ... colocar um margin ou um padding certinho

leticiavargas commented on src/components/productCard/index.jsx:
```diff
@@ -0,0 +1,26 @@
+          {rating} avaliações
+        </p>
+        <p className="cardPrice">
+          R$ {price.toLocaleString("pt-BR", { minimumFractionDigits: 2 })}
```
é bom padronizar o uso ... aqui está usando o toLocaleString ... no outro arquivo um NumberFormat ... vê qual atende melhor é padroniza.

leticiavargas commented on src/components/productCard/styles.css:
```diff
@@ -0,0 +1,99 @@
+.cardDescription,
+.cardRating,
+.cardPrice {
+  font-family: "Roboto";
```
Todo app é Roboto? Se for, vale a pena definir a font-family de maneira global. Outro ponto, é sempre bom colocar o sans-serif, pois caso o navegador não consiga usar a Roboto ele irá aplicar uma font sem serifa disponível

`font-family: 'Roboto', sans-serif;`

leticiavargas commented on src/components/starRating/index.jsx:
```diff
@@ -0,0 +1,25 @@
+    <Rating
+      name="Avaliação do produto"
+      value={rating}
+      getLabelText={(value) => `${value} ${value > 1 ? "Corações" : "Coração"}`}
```
ueh ... pq corações?

LuanRethink committed:
name changed on screenreader text label

LuanRethink committed:
changed alt text

LuanRethink committed:
removed unwanted spacing

mhid-dev committed:
Collapsible Component v1 (#20)

* Collapsible Section v1

* Atualizando Collapsible com comentários do PR

LuanRethink committed:
Merge branch 'showcase' into LB-carousel-gallery-images

MarcosPauloRTK committed:
Code correction #1

Changes made to the component .JSX and CSS considering previous code reviews on the onClick and the logic itself.

LuanRethink committed:
implemented cardImage at card and carousel

LuanRethink committed:
removed

LuanRethink committed:
Merge pull request #16 from leticiavargas/LB-carousel-gallery-images

Lb carousel gallery images

mhid-dev committed:
Alterações para bater com dados do backend

vinicius-delmo committed:
refactor: removed usestate, changed app.css, changed button to label

vinicius-delmo committed:
Merge branch 'showcase' into vd-Filter_Product

vinicius-delmo committed:
Merge pull request #8 from leticiavargas/vd-Filter_Product

feat(filter): Add product filter select component

vinicius-delmo committed:
Merge pull request #21 from leticiavargas/vd-Search-Product

vd search product

mhid-dev committed:
Merge branch 'showcase' into mp-suggestion_card
